homeassistant/components/imou_life/__init__.py

```diff
@@ -0,0 +1,72 @@
+"""Support for Imou camera"""
```

Comment:
Is this integration specifically for Imou's cameras?

If so, why does this need a cloud-dependent integration? I have a few Imou cameras in my test cabinet, and they work with ONVIF directly and locally.

../Frenck

Reply:
Hello,the imou integration is specifically developed for imou cameras.
For example,if people and devices are not together,you can remotely operate the devices under your own account through integration in the cloud,which is more convenient then local ONVIF.

> Is this integration specifically for Imou's cameras?
> 
> If so, why does this need a cloud-dependent integration? I have a few Imou cameras in my test cabinet, and they work with ONVIF directly and locally.
> 
> ../Frenck